### PR TITLE
March 2024 Marketplace Guides

### PR DESCRIPTION
**Pull request by tbaka**
Adds LinuxGSM and Passbolt guides.

### CODE REVIEW

File: docs/products/tools/marketplace/guides/passbolt/index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+* **Use TLS?**: No
+* **Port**: 25
+* **SMTP client** leave blank
+* **Sender name**: root
+* **Sender email**: root@<example.com>
+
+{{< note >}}
+To make the most out of Passbolt you need a working email setup for email notifications (e.g. - account registration, password recovery and other critical notifications). For more information on setting this up, see the [Configure Email Providers](https://help.passbolt.com/configure/email/setup) page on Passbolt's website.
+
+Regardless of how you configure your mail server, it is strongly recommended that you follow the best practices to ensure mail deliverability. Please see the [Running a Mail Server](https://www.linode.com/docs/guides/running-a-mail-server/) guide for more information.
```
Comment (jacabral) [inline docs/products/tools/marketplace/guides/passbolt/index.md:80]: Please avoid recommending anything to our customers.

Suggest changing "It is strongly recommended" to "for the desired results, follow the best practices to ensure mail deliverability."

File: docs/products/tools/marketplace/guides/linuxgsm/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+{{< content "marketplace-verify-standard-shortguide">}}
+
+{{< note >}}
+**Estimated deployment time:** LinuxGSM should be fully installed within 10-15 minutes after the Compute Instance has finished provisioning.
+{{< /note >}}
+
+## Configuration Options
+
+- **Supported distributions:** Ubuntu 22.04 LTS
+- **Recommended plan:** All plan types and sizes can be used.
```
Comment (jacabral) [inline docs/products/tools/marketplace/guides/linuxgsm/index.md:29]: Replace "Recommended" with "Suggested"

File: docs/products/tools/marketplace/guides/passbolt/index.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+{{< content "marketplace-verify-standard-shortguide">}}
+
+{{< note >}}
+**Estimated deployment time:** Passbolt should be fully installed within 5-10 minutes after the Compute Instance has finished provisioning.
+{{< /note >}}
+
+## Configuration Options
+
+- **Supported distributions:** Ubuntu 22.04 LTS
+- **Recommended plan:** We recommend a 4GB Dedicated CPU or Shared Compute instance for Passbolt.
```
Comment (jacabral) [inline docs/products/tools/marketplace/guides/passbolt/index.md:29]: Per our legal department, we don't recommend anything to our customers. 

Replace "Recommended" with "Suggested"
Replace "We recommend" with "For the best results, use a 4GB Dedicated CPU or Shared Compute instance for Passbolt."

File: docs/products/tools/marketplace/guides/passbolt/index.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+You will need to create a passphrase to access Passbolt 
+
+    ![Screenshot of Passbolt passphrase](passphrase_screen.jpg)
+
+Once the passphrase is set you will be able to view the Passbolt main screen. 
+
+    ![Screenshot of Passbolt main screen](passbolt_mainscreen.jpg)
+
+Please visit [Passbolt Installation Documentation](https://help.passbolt.com/hosting/install/ce/ubuntu/ubuntu.html) for information on how to configure Passbolt.
```
Comment (jacabral) [inline docs/products/tools/marketplace/guides/passbolt/index.md:63]: Suggest:

"For information on how to configure Passbolt, visit [Passbolt Installation Documentation](https://help.passbolt.com/hosting/install/ce/ubuntu/ubuntu.html)."

**Review by jacabral — APPROVED** [review]
Please see my suggested edits.

File: docs/products/tools/marketplace/guides/linuxgsm/index.md (hunk 2 above)
Comment (tbaka) [reply docs/products/tools/marketplace/guides/linuxgsm/index.md:29]: Every single marketplace app guide uses the phrase "recommended plan." I can create another PR to change if necessary.

Comment (tbaka) [reply docs/products/tools/marketplace/guides/linuxgsm/index.md]: We discussed this and determined it is not necessary to retroactively change the language "recommended plan" in existing guides.

**Review by jacabral — APPROVED** [review]

**Pull request merged**